app/src/androidTest/java/org/mozilla/fenix/ui/robots/BrowserRobot.kt

```diff
@@ -366,7 +366,7 @@ class BrowserRobot {
         }
 
         fun openNavigationToolbar(interact: NavigationToolbarRobot.() -> Unit): NavigationToolbarRobot.Transition {
-
+            mDevice.waitForIdle(waitingTime)
```

Comment:
is there a reason why you're not using waitNotNull instead?  I'm going to merge this because of all the failures happening, but we should change this later if this isn't what's desired here